content/blog/configuring-sso-for-hpe-greenlake-central-private-cloud-enterprise-and-hpe-greenlake-glcp-using-okta.md

```diff
@@ -0,0 +1,166 @@
+
+**S﻿teps to be followed in Customer Okta:**
+
+###   Configure SSO/SAML applications in Okta    
+
+   To configure application metadata in Okta, complete the following steps:
+
+* Step 1: Create an Okta SAML application
+* Step 2: Configure Sign On settings
+* Step 3: Export the SAML 2.0 IdP metadata
+* Step 4: Configure the SAML connection in HPE GreenLake Cloud Platform
+
+**Step 1: Create an Okta SAML application**
+
+1. Log in to the Okta administration console.
+2. Click **Applications > Create new app integration.** The Create a new app integration window opens.
+3. Select SAML 2.0 and click **Next**.
+
+![](/img/ws-image0.png)
+
+Provide a name for the SAML application which gets connected to HPE GreenLake Cloud Platform(GLCP)
+
+![](/img/customer_saml_app.jpg)
+
+**Step 2: How to configure Single Sign On settings**
+
+1. Enter the SAML information.
+
+   Under General:
+
+    **Single Sign on URL:** https://sso.common.cloud.hpe.com/sp/ACS.saml2
+
+    **Audience URI (SP Entity ID):** https://sso.common.cloud.hpe.com
+
+    **Name ID format EmailAddress**
+
+    **Application username Email**
+
+    **NameID = user.email**
+
+    **gl_first_name = user.FirstName**
+
+    **gl_last_name = user.LastName**
+
+    **hpe_ccs_attribute = (See Below)**
+
+   See here for IdP attribute details: <https://support.hpe.com/hpesc/public/docDisplay?docId=a00120892en_us>
+
+   A new SAML attribute has been added “hpe_ccs_attribute” which tells HPE GreenLake and HPE Greenlake Central (Private Cloud Enterprise) the exact role/permissions for each user. The following describes how to format the attribute.
+
+   Format: {version}#{pcid}:{app id}:{role_name}:{ALL_SCOPES}
+
+![](/img/hpe-greenlake-saml-attributes.jpg)
+
+![](/img/workspace-pcid.jpg)
+
+![](/img/glp_role_name.jpg)
+
+The **hpe_ccs_attribute** always starts with version*1#. You must first configure the attributes for HPE GreenLake and To do so, enter the PCID for the account, followed by the HPE GreenLake application ID. This will always be **00000000-0000-0000-0000-000000000000**. Following this, enter the role name and ALL*SCOPES**.** 
+
+Example:
+
+version_1#7ede5c36b7b911edacf45a78eb8b07d1:00000000-0000-0000-0000-000000000000:Observer:ALL_SCOPES
+
+![](/img/saml_settings.jpg)
+
+2﻿. Complete the setup.
+
+![](/img/ws-image7.png)
+
+Click Next and Select “Internal App”, then Finish.
+
+**Step 3:** **Export the SAML 2.0 IdP metadata**
+
+1. Click Next – Configure the Sign On settings
+
+   You will find two options are available: **View Setup Instructions** which steps you through the SAML configuration and **Identity Provider metadata**, which will produce an XML file that can be loaded into HPE GreenLake Cloud Platform (GLCP) application
+
+   Suggestion: Click **Identity Provider metadata** and save the XML data to a file.
+
+   ![](/img/ws-image9.png)
+2. C﻿lick Next.       
```

Comment:
Something is strange with the formatting of bullets 2 and 3. You might also want to bold the terms "Next", "Internal app" and "Finish". The word "Click" should be "click".

In other words:
2. Click **Next**
3. Select **Internal app** and click **Finish**.

![image](https://github.com/hpe-dev-incubator/hpe-dev-portal/assets/77496261/ebbaa93e-1514-415b-9a24-60626612d55d)